hera_mc/mc_session.py

```diff
@@ -1609,6 +1831,8 @@ def correlator_control_command(self, command, starttime=None, duration=None,
         overwrite_take_data: boolean
             only applies if command is 'take_data': If there is already a take data starttime
             in the future, overwrite it with this command.
+        config_file: string file name
```

Comment:
Make explicit that this is the full path to the file. And that it needs to be a path that the correlator can find.